### PR TITLE
Add messages and basic shoprotation command

### PR DESCRIPTION
**Pull request by NYC1809**

### CODE REVIEW

File: src/main/kotlin/de/skycave/shoprotation/Enums/Message.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    NO_PLAYER("&cDieser Befehl ist nur für Spieler."),
+
+    ;
+
+    fun getFormatted(): String {
+        return ChatColor.translateAlternateColorCodes('&', message)
+    }
+
+    fun getMessage(): String {
+        return ChatColor.translateAlternateColorCodes('&', PREFIX.message + message)
```
Comment (heuerleon) [inline src/main/kotlin/de/skycave/shoprotation/Enums/Message.kt:20]: This is how we used to return messages in older plugins. Where did you take that from?

**Review by heuerleon — CHANGES_REQUESTED** [review]

File: src/main/kotlin/de/skycave/shoprotation/Enums/Message.kt (hunk 1 above)
Comment (NYC1809) [reply src/main/kotlin/de/skycave/shoprotation/Enums/Message.kt:20]: I copied that part from MiniEvents. I will change that to the newer version.

Comment (heuerleon) [reply src/main/kotlin/de/skycave/shoprotation/Enums/Message.kt:20]: You can look here for reference: https://github.com/skycavemc/chest-refiller/blob/main/src/main/kotlin/de/skycave/chestrefiller/enums/Message.kt and here https://github.com/skycavemc/chest-refiller/blob/main/src/main/kotlin/de/skycave/chestrefiller/models/ChatMessage.kt

**Comment by heuerleon** [comment]
Looks good!

**Review by heuerleon — APPROVED** [review]

**Pull request merged**